src/js/gi/containers/SearchPage.jsx

```diff
@@ -111,8 +109,6 @@ export class SearchPage extends React.Component {
     delete query.page;
 
     const shouldRemoveFilter =
-      (field !== 'caution' && !queryValue) ||
-      (field === 'caution' && queryValue) ||
```

Comment:
There should still be a condition here for the checkbox true/false values, I think.

`const shouldRemoveFilter = !value || (...)`